pkg/planner/core/generator/shallow_ref/shallow_ref_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/pingcap/tidb/pkg/util"
+)
+
+func TestHash64Equals(t *testing.T) {
+	updatedCode, err := GenShallowRef4LogicalOps()
+	if err != nil {
+		t.Errorf("Generate XXXShallowRef code error: %v", err)
+		return
+	}
+	currentCode, err := os.ReadFile("../../operator/logicalop/shallow_ref_generated.go")
```

Comment:
It needs to update the bazel config and register this file to make it easy to find.

Reply:
You can copy the config like this ```pkg/planner/core/BUILD.bazel:337```